crates/core/tests/helpers/constants.rs

```diff
@@ -0,0 +1,4 @@
+pub const EVM_ADDRESS: &str = "0x132d200e5ceedf1a4634eee081cfeb077d92e4fd";
```

Comment:
I think we can use lazy_static! here to make those constants directly the desired type, we've done it once already in the codebase

Reply:
and `EVM_ADDRESS` should be derived from the `EVM_PRIVATE_KEY` to prevent any mismatch